package.json

```diff
@@ -5,6 +5,8 @@
   "dependencies": {
     "@babel/core": "7.2.2",
     "@microsoft/microsoft-graph-client": "^1.5.2",
+    "@svgr/webpack": "4.1.0",
+    "@uifabric/styling": "^6.46.0",
```

Comment:
should we call an exact package here?

Reply:
Yes we should. Thanks for pointing this out.